libraries/rush-lib/src/api/packageManager/PnpmPackageManager.ts

```diff
@@ -22,6 +22,14 @@ export class PnpmPackageManager extends PackageManager {
    */
   public readonly pnpmfileFilename: string;
 
+  /**
+   * The filename of the shrinkwrap file of a subspace that is used by the package manager
+   *
+   * @remarks
+   * Example: `.pnpmfile-<subspace_name>.cjs
+   */
+  public subspacePnpmfileFilename: (subspaceName: string) => string;
```

Comment:
Why is this a dynamic function?

The RFC said:
![image](https://github.com/microsoft/rushstack/assets/4673363/773e24cf-af37-43a1-8113-288b8cd086bc)

Reply:
Removed the dynamic function as it's just going to use the same name as the regular pnpmfile